src/Main.elm

```diff
@@ -387,19 +461,24 @@ viewAnswersSection model =
     , viewAnswerList model
     ]
 
-viewQuestion : Question -> Html msg
+viewQuestion : Question -> Html Msg
 viewQuestion ques =
+  let
+      stringId =
+        ques.q_id
+        |> String.fromInt
+  in
          tr []
-            [ td [ class "td-check"] [ input [type_ "checkbox", value "rice" ] [] ]
+            [ td [ class "td-check"] [ input [type_ "checkbox", value stringId, onInput QuestionChecked ] [] ]
```

Comment:
does this still work if a question gets unchecked? i.e. does its id get removed from the list?

Reply:
looking at the code above it looks like it would work :+1:

Reply:
Yes this should, since I filter the entrant